match/lib/match/nuke.rb

```diff
@@ -274,31 +286,62 @@ def delete_files!
     def certificate_type(type)
```

Comment:
Do we need to handle `DEVELOPER_ID_KEXT` and `DEVELOPER_ID_APPLICATION` at all, here? (not sure what these 2 are for)

Reply:
`DEVELOPER_ID_APPLICATION` cannot be revoked and unsure if `DEVELOPER_ID_KEXT` can be revoked but we can handle that later once I get kext access in one of my accounts 🙃